### PR TITLE
docs(readme): reorg readme and contributing

### PR DESCRIPTION
**Pull request by kumikokashii**
This update goes together with monorepo's readme & contributing update https://github.com/HospitalRun/hospitalrun/pull/381

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,147 +1,89 @@
...
-maybe you'd like to open a pull request to address one of [good-first-issue](https://github.com/HospitalRun/hospitalrun-frontend/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+## 1. Check out the [HospitalRun General Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CONTRIBUTING.md) for how to:
+   1. Browse Issues
+   2. Get Assigned an Issue
+   3. Fork the Repository and Create a Branch
+   4. Commit Changes
+   5. Submit a Pull Request
 
-Make sure to checkout the [HospitalRun Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CONTRIBUTING.md)
+## 2. After creating a local branch (step 3 above), follow these steps to configure CouchDB
```
Comment (jackcmeyer) [inline .github/CONTRIBUTING.md:12]: I think we should change to `After creating a local fork...`

File: README.md [hunk 2]
```diff
@@ -1,125 +1,27 @@
-# HospitalRun Frontend
```
Comment (jackcmeyer) [inline README.md:1]: I think we should keep the title of the page and add a short description of what the repository is underneath the title

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,147 +1,89 @@
...
 
-To contribute,
-[fork](https://help.github.com/articles/fork-a-repo/) this repo, commit your changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
+CouchDB is the server side database which data from the frontend will sync to. CouchDB is required to login
+to HospitalRun. You could install and run CouchDB in any way you wish. For convienence, we have added a docker compose file in the
+root of this project to launch CouchDB. Below is the steps:
```
Comment (jackcmeyer) [inline .github/CONTRIBUTING.md:16]: "Below is the steps:" --> "Below are the steps:"

File: README.md [hunk 4]
```diff
@@ -1,125 +1,27 @@
...
 
-**Version 1.0.0-beta is no longer supported. Version 2 is currently under development.**
-
-- To contribute, follow the guidelines in the readme or alternatively ask for details on Slack channel [#contributors](https://hospitalrun-slack.herokuapp.com).
-- To use version 1.0.0-beta (not production ready) in a hospital facility, ask for support on Slack channel [#troubleshooting](https://hospitalrun-slack.herokuapp.com).
-
 <div align="center">
 
-[![Slack](https://img.shields.io/badge/Slack-Join%20our%20devs%20group-blueviolet?style=for-the-badge&logo=slack)](https://hospitalrun-slack.herokuapp.com)
+![Status](https://img.shields.io/badge/Status-developing-brightgree) [![Release](https://img.shields.io/github/release/HospitalRun/hospitalrun-frontend.svg)](https://github.com/HospitalRun/hospitalrun-frontend/releases) [![Version](https://img.shields.io/github/package-json/v/hospitalrun/hospitalrun-frontend)](https://github.com/HospitalRun/hospitalrun-frontend/releases)
```
Comment (jackcmeyer) [inline README.md:20]: Typically I see badges above the content

File: README.md [hunk 5]
```diff
@@ -1,125 +1,27 @@
...
-
-[![Contributors](https://opencollective.com/hospitalrun/contributors.svg?width=960&button=false)](https://github.com/HospitalRun/hospitalrun-frontend/graphs/contributors)
-
-## Founders
-
-<!-- prettier-ignore -->
-| [<img src="https://avatars0.githubusercontent.com/u/609052?s=460&v=4" width="100px;"/><br /><sub><b>John Kleinschmidtr</b></sub>](https://github.com/jkleinsc) | [<img src="https://avatars0.githubusercontent.com/u/929261?s=400&v=4" width="100px;"/><br /><sub><b>Joel Worrall</b></sub>](https://github.com/tangollama)  | [<img src="https://avatars0.githubusercontent.com/u/1319791?s=460&v=4" width="100px;"/><br /><sub><b>Joel Glovier</b></sub>](https://github.com/jglovier)  |
-|---|---|---|
-
-# License
```
Comment (jackcmeyer) [inline README.md:123]: License should stay in the README IMO

File: README.md [hunk 6]
```diff
@@ -1,125 +1,27 @@
...
-
-In order to optimize the workflow and to prevent multiple contributors working on the same issue without interactions, a contributor must ask to be assigned to an issue by one of the core team members: it's enough to ask it inside the specific issue.
-
-## How to commit
-
-This repo uses Conventional Commits. Commitizen is mandatory for making proper commits. Once you have staged your changes, can run `npm run commit` from the root directory in order to commit following our standards.
-
-<hr />
-
-# Behind HospitalRun
```
Comment (jackcmeyer) [inline README.md:69]: I like the idea of keeping this section on each project. 

Contributors are collected by repo, so for everyone to be recognized, I think we should keep this. Plus visibility for this section on all repos is a good thing IMO.

File: README.md (hunk 5 above)
Comment (tehKapa) [reply README.md:123]: +1

File: README.md (hunk 2 above)
Comment (tehKapa) [reply README.md:1]: +1

File: README.md [hunk 7]
```diff
@@ -1,125 +1,27 @@
-# HospitalRun Frontend
+[Visit this page for general information on the HospitalRun application](https://github.com/HospitalRun/hospitalrun/blob/master/README.md) including:
 
-<div align="center">
```
Comment (tehKapa) [inline README.md:3]: You need to restore this header with the badges

File: README.md [hunk 8]
```diff
@@ -1,125 +1,27 @@
...
-![dependabot](https://api.dependabot.com/badges/status?host=github&repo=HospitalRun/hospitalrun-frontend) [![Slack](https://hospitalrun-slack.herokuapp.com/badge.svg)](https://hospitalrun-slack.herokuapp.com) [![Run on Repl.it](https://repl.it/badge/github/HospitalRun/hospitalrun-frontend)](https://repl.it/github/HospitalRun/hospitalrun-frontend)
-[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/HospitalRun/hospitalrun-frontend)
+How can I deploy 1.0.0-beta? | Where do I report a bug or request a feature? | How can I contribute? (There are several other ways besides coding) | What is the project structure? | What is the application infrastructure? | Who is behind HospitalRun? etc.
 
-</div>
+# How to Contribute to the Frontend Repository
 
-React frontend for [HospitalRun](http://hospitalrun.io/): free software for developing world hospitals.
```
Comment (tehKapa) [inline README.md:13]: It's to figure out what it's all about. To restore

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (fox1t) [reply .github/CONTRIBUTING.md:16]: "CouchDB is the server side database which data from the frontend will sync to" -> CouchDB is the server side database which data from the frontend will sync from and to.

File: README.md (hunk 5 above)
Comment (fox1t) [reply README.md:123]: I agree on this: is has to be clear at first glance.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (kumikokashii) [reply .github/CONTRIBUTING.md]: ✅

File: README.md (hunk 2 above)
Comment (kumikokashii) [reply README.md]: ✅

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (kumikokashii) [reply .github/CONTRIBUTING.md]: ✅

File: README.md (hunk 4 above)
Comment (kumikokashii) [reply README.md]: ✅

File: README.md (hunk 5 above)
Comment (kumikokashii) [reply README.md]: ✅

File: README.md (hunk 6 above)
Comment (kumikokashii) [reply README.md:69]: ✅

File: README.md (hunk 7 above)
Comment (kumikokashii) [reply README.md]: ✅

File: README.md (hunk 8 above)
Comment (kumikokashii) [reply README.md]: ✅

**Comment by kumikokashii** [comment]
Feedback has been incorporated, thank you! Errors seem to be from places unrelated to this PR?

**Comment by tehKapa** [comment]
Yes, it doesn't depend on PR 👯‍♂️

File: README.md [hunk 9]
```diff
@@ -14,61 +14,22 @@ React frontend for [HospitalRun](http://hospitalrun.io/): free software for deve
...
-Use the standards in [this readme](https://github.com/HospitalRun/hospitalrun-frontend/tree/master/src/locales/README.md).
-
-## Online one-click setup for contributing
-
-Contribute to HospitalRun using a fully featured online development environment that will automatically: clone the repo, install the dependencies and start the webserver.
-
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/HospitalRun/hospitalrun-frontend)
-
-## Running Tests and Linter
+How can I deploy 1.0.0-beta? | Where do I report a bug or request a feature? | How can I contribute? (There are several other ways besides coding) | What is the project structure? | What is the application infrastructure? | Who is behind HospitalRun? etc.
```
Comment (jackcmeyer) [inline README.md:21]: I think this should be a bulleted list to make it easier to read

Comment (tehKapa) [reply README.md:21]: Done

Comment (jackcmeyer) [reply README.md:21]: I don't see the changes 😀

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -61,93 +44,61 @@ To delete the development database, go to the root of the project and run `docke
 
 ## Online one-click setup for contributing
 
-Contribute to HospitalRun using a fully featured online development environment that will automatically: clone the repo, install the dependencies and start the webserver.
-
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/HospitalRun/hospitalrun-frontend)
+Open a file named `.env` and add a line `REACT_APP_HOSPITALRUN_API=http://localhost:5984` if you don't have it yet.
```
Comment (jackcmeyer) [inline .github/CONTRIBUTING.md:48]: This section's header is not correct. This information should go under `Start the application`

Comment (tehKapa) [reply .github/CONTRIBUTING.md:48]: @kumikokashii shall I do it?

Comment (fox1t) [reply .github/CONTRIBUTING.md:48]: I am going to merge this PR then make the change

**Comment by fox1t** [comment]
LGTM

**Pull request merged**